app/src/main/res/layout/welcome_slide1.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Instead of writing 4 layouts, can we make just one layout and make 3 arrays: 
1. one for storing title -> title[ ] = {"Login", "Chat Interface", "Skill Listing", "Settings"}
2. second for images -> images[ ] = {R.drawable.img1, .... }
3. third for description -> desc[ ] = {"desc1", ...}

and use this array to set up the layout. I am not sure if we can do this. Can you try this once, please.

Reply:
Okay I'll try 👍

Reply:
@chiragw15 I tried doing this but was not able to implement it, I am sorry. Also what other changes do i need to make in this PR?